### ISSUE TITLE
Unable to scroll stacked tabs horizontally when card layout enabled

### ISSUE THREAD

**Issue by alfierrc** [Akifyss/obsidian-border#276]
It's entirely possible I'm missing a setting somewhere, but when card layout is enabled the ability to scroll stacked tabs horizontally seems to break. I am able to click the headers of each slide to bring it to the front but not scroll between them. See gif below:

![obsidian-border2](https://github.com/Akifyss/obsidian-border/assets/59963143/03a0a56d-9e8c-4edf-9be5-b225b9c7fcce)

Thanks!

**Comment by Akifyss** [comment]
Sorry, this issue cannot be perfectly resolved, but there are some compromise solutions. Please refer to this issue: https://github.com/Akifyss/obsidian-border/issues/47

**Comment by alfierrc** [comment]
Apologies I should have looked harder! Thanks for pointing me in the right direction, love the theme.

**Comment by tolliko** [comment]
hey @Akifyss ! thanks for the beautiful theme, love it! However, horizontal scroll it pretty essential in my workflow, so I had to switch to another theme. So, I switched to minimal, but missed so much for your gorgeous theme! And I decided to make cards view in that minimal with snippets. I'm not strong with css, but I've managed to do smth similar to border theme (not so powerful, but it works)

BUT, why am I writing all this: I've found solution to the horizontal scroll! I've checked it with Border theme and seems it works just fine! I even had to register on github to share this fix! But as I said - i'm bad at css, maybe that is a shitty solution

so:

```
/* just override disabled scroll */
body:not(.is-mobile).theme-dark.card-layout-open-dark .workspace-tab-container {
    overflow-x: scroll !important;
}

/* hide scrollbar (scroll still works) */
.workspace .mod-root .workspace-tabs.mod-stacked ::-webkit-scrollbar:horizontal {
    display: none;
}
```


hope it helps

thanks again for your efforts and good luck with developing great things
